test/drafts/ERC20Migrator.test.js

```diff
@@ -44,6 +44,40 @@ contract('ERC20Migrator', function ([_, owner, recipient, anotherAccount]) {
       });
     });
 
+    describe('before migration', function () {
```

Comment:
```suggestion
    context('before starting the migration', function () {
```

We may want to emphasize that the issue here is that the migration was not properly started.

Reply:
I agree - good catch. I really like that you take these details seriously 😄. I have updated the PR.